mlkem/cbmc.h

```diff
@@ -118,7 +118,7 @@
                          value_lb, value_ub)                           \
   __CPROVER_forall                                                     \
   {                                                                    \
-    unsigned qvar;                                                     \
+    __CPROVER_size_t qvar;                                             \
```

Comment:
I wonder what the performance would be with `__CPROVER_ssize_t` instead, which would avoid a type cast (at present, CBMC uses signed integers for index expressions (the C standard is not clear as whether index expressions have signed or unsigned integer type)).

Reply:
I tried CPROVER_ssize_t and it makes no difference - still much slower. It seems the difference is something to do with "unsigned" being "32 bit" but ssize_t and size_t are "64 bit". Look at the generated GOTO files.